src/AmbientSounds.Uwp/App.xaml.cs

```diff
@@ -37,6 +38,27 @@ public App()
                 // Ref: https://docs.microsoft.com/en-us/windows/uwp/xbox-apps/how-to-disable-mouse-mode
                 this.RequiresPointerMode = ApplicationRequiresPointerMode.WhenRequested;
             }
+
+            object themeObject = ApplicationData.Current.LocalSettings.Values["themeSetting"];
+            if (themeObject != null)
+            {
+                string theme = themeObject.ToString();
+                switch (theme)
+                {
+                    case "light":
+                        App.Current.RequestedTheme = ApplicationTheme.Light;
+                        break;
+                    case "dark":
+                        App.Current.RequestedTheme = ApplicationTheme.Dark;
+                        break;
+                    default:
+                        break;
+                }
+            }
+            else
+            {
+                ApplicationData.Current.LocalSettings.Values["themeSetting"] = "default";
+            }
```

Comment:
Is it possible to move this to a private method? That way, the constructor stays concise and readable.